src/test/java/org/fundacionjala/trello/ui/hooks/CommonHooks.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.trello.ui.hooks;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import org.fundacionjala.trello.core.utils.WebDriverManager;
+
+public class CommonHooks {
```

Comment:
Avoid having this class, when you have generic classes it ends having a lot of hooks that looked common